src/components/posterior_component/posterior.sh

```diff
@@ -100,6 +100,22 @@ run_posterior() {
         source ${GEOSChemEnv}
     fi
 
+    if "$OptimizeBCs"; then
+        if "$KalmanMode"; then
+            period_i=$((period_i+1))
```

Comment:
Maybe it's obvious but why are we doing `period_i + 1` ?

Reply:
Yeah this is definitely a bug! Not sure why I put this in -- Nice catch!